src/app/images/components/ImagesTable/DeleteImageConfirm/DeleteImageConfirm.test.tsx

```diff
@@ -69,15 +69,19 @@ describe("DeleteImageConfirm", () => {
       }),
     });
     const store = mockStore(state);
-    const wrapper = mount(
+    render(
```

Comment:
By the way, I updated the render methods so you can pass the store instead of having a wrapping provider for these kinds of tests where you need to access the dispatched actions.

You can see a test that uses it here: https://github.com/canonical/maas-ui/pull/4372/files#diff-3efe7df819d51643df4e265bac2a50f07461c86da7bb7e5b60edc4c3703bfeaaR21.

Reply:
YEEEES!!!! Ahh this is gonna make so many things easier

Reply:
Getting some weird linting errors with using it. The tests run absolutely fine, I think it's something to do with the type definitions: 
![image](https://user-images.githubusercontent.com/35104482/187681665-9c0da3ef-9048-4d99-a8c5-37a1da736ab8.png)

Reply:
Yeah pushing a commit with everything changed like this fails the build because of this error. I've pushed another commit reverting it, will await your review in the morning

Reply:
Ah yeah there's one other thing you need to do which is to set the state type when you set up the mock store, like [this](https://github.com/canonical/maas-ui/pull/4372/files#diff-3efe7df819d51643df4e265bac2a50f07461c86da7bb7e5b60edc4c3703bfeaaR15): `const mockStore = configureStore<RootState, {}>();`

In your file I think that's here: https://github.com/canonical/maas-ui/pull/4374/files/7c8ed3364041c71fc9f0344c8e2fda93aaf387c6#diff-1508b10c1565d92ae3052827344be438f2d9af8f303abaac4f808b5e7f886eb3L16.

Reply:
That worked like a charm, thanks!